lib/include/openamp/rpmsg_virtio.h

```diff
@@ -75,8 +75,26 @@ struct rpmsg_virtio_device {
 	struct rpmsg_virtio_shm_pool *shpool;
 };
 
-#define RPMSG_REMOTE	VIRTIO_DEV_SLAVE
-#define RPMSG_MASTER	VIRTIO_DEV_MASTER
+#define RPMSG_REMOTE	VIRTIO_DEV_DEVICE
+#define RPMSG_HOST	VIRTIO_DEV_DRIVER
+
+#ifdef VIRTIO_DEV_MASTER
+#if defined(__GNUC__)
+_Pragma("GCC warning\"Please use VIRTIO_DEV_DRIVER instead of VIRTIO_DEV_MASTER which is deprecated\"")
+#elif defined(__ICCARM__)
+#define DO_PRAGMA(x) _Pragma(#x)
+DO_PRAGMA(message("Warning: Please use VIRTIO_DEV_DRIVER instead of VIRTIO_DEV_MASTER which is deprecated"))
+#endif
+#endif
+
+#ifdef VIRTIO_DEV_SLAVE
+#if defined(__GNUC__)
+_Pragma("GCC warning\"Please use VIRTIO_DEV_DEVICE instead of VIRTIO_DEV_SLAVE which is deprecated\"")
+#elif defined(__ICCARM__)
+#define DO_PRAGMA(x) _Pragma(#x)
+DO_PRAGMA(message("Warning: Please use VIRTIO_DEV_DEVICE instead of VIRTIO_DEV_SLAVE which is deprecated"))
+#endif
+#endif
```

Comment:
To also limit message size here in same way
In parallel I will send a PR to increate the line length limit to 100 ( as done in Linux kernel)